hi.py

```diff
@@ -0,0 +1,8 @@
+def hello():
+  return "Hello User" 
+
+    
+    
+    
+if name == "__name___"
```

Comment:
<a href="https://www.pantomax.co" target="_blank"><img src="https://raw.githubusercontent.com/pantomaxdotco/assets/refs/heads/main/pr-comment-branding.png" width="50"></a>
Correct the condition to `if __name__ == "__main__":`.

```py
if __name__ == "__main__":
```